### PR TITLE
[Dashboard] Remove use-scroll-position

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR primarily focuses on removing the dependency on `@n8tb1t/use-scroll-position` and refactoring the `HomepageTopNav` component in `Topnav.tsx` to utilize `@chakra-ui/react` for styling and layout.

### Detailed summary
- Removed `@n8tb1t/use-scroll-position` dependency from `package.json`.
- Eliminated the `useScrollPosition` hook from `HomepageTopNav`.
- Introduced `Box`, `Container`, and `Flex` components from `@chakra-ui/react` for layout.
- Adjusted the `bgColor` and `backdropFilter` properties in the `Box` component.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
* **#5274** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5274?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/components/product-pages/common/Topnav.tsx [hunk 1]
```diff
@@ -1,43 +1,20 @@
-import { Box, Container, Flex, useBreakpointValue } from "@chakra-ui/react";
-import { useScrollPosition } from "@n8tb1t/use-scroll-position";
```
Comment (kien-ngo) [inline apps/dashboard/src/components/product-pages/common/Topnav.tsx:2]: This navbar is only used in the pages that (pretty sure) will be converted to Framer.

What it does is change the background color effects on scrolling down. It not that big of a deal and I think we can just make it "static".

File: apps/dashboard/src/components/product-pages/common/Topnav.tsx [hunk 2]
```diff
@@ -1,43 +1,20 @@
...
       left={0}
       w="100%"
-      willChange="backdrop-filter background-color"
       zIndex="overlay"
       as="header"
-      bgColor={isScrolled ? "rgba(0,0,0,0.8)" : "rgba(0,0,0,0)"}
-      backdropFilter={
-        isScrolled ? "saturate(180%) blur(20px)" : "saturate(100%) blur(0px)"
-      }
+      bgColor="rgba(0,0,0,0)"
```
Comment (MananTank) [inline apps/dashboard/src/components/product-pages/common/Topnav.tsx:16]: this is not looking good when you scroll the page down

**Review by MananTank — CHANGES_REQUESTED** [review]
address comments

File: apps/dashboard/src/components/product-pages/common/Topnav.tsx (hunk 2 above)
Comment (kien-ngo) [reply apps/dashboard/src/components/product-pages/common/Topnav.tsx:16]: ah

Comment (kien-ngo) [reply apps/dashboard/src/components/product-pages/common/Topnav.tsx:16]: @MananTank I gave it a default blur-20px. 



This page is very soon to be deleted by so it's up to you. I'm fine with closing this PR

https://www.notion.so/thirdweb/Pages-that-should-move-into-framer-or-be-removed-12e8141725338051a965daa7ae2e60dc

Comment (jnsdls) [reply apps/dashboard/src/components/product-pages/common/Topnav.tsx:16]: yeah let's not do this - also let's not waste time on pages that will be deleted or moved to framer soon, we have lots to do on "in app" pages :)

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/components/product-pages/common/Topnav.tsx (hunk 2 above)
Comment (kien-ngo) [reply apps/dashboard/src/components/product-pages/common/Topnav.tsx:16]: Understood ^

**Pull request closed without merging**